__tests__/resource/csharp-nunit-input.json

```
@@ -0,0 +1,289 @@
+{
+  "sentAt": {
+    "low": 1105186912,
+    "high": 395,
+    "unsigned": true
+  },
+  "serverInfo": {
+    "apiUrl": "https://api.kobiton.com",
+    "portalUrl": "https://portal.kobiton.com",
+    "username": "kobitonadmin+appiumScriptGen",
+    "apiKey": "your_kobiton_api_key"
+  },
+  "isManualSession": true,
+  "manualSessionId": "605",
+  "devices": [
+    {
+      "id": 1881,
+      "name": "Lenovo Tab M10 FHD Plus",
+      "capabilities": {
+        "platformName": "Android",
+        "platformVersion": "10",
+        "resolution": {
+          "width": 720,
+          "height": 1600,
+          "scale": 1
+        }
+      }
+    }
+  ],
+  "testSteps": [
+    {
+    "id": "17000",
+      "context": "NATIVE",
+      "selectorConfigurations": [
+        {
+          "selectors": [
+            {
+              "type": "xpath",
+              "value": "//*[@resource-id='com.android.chrome:id/url_bar']"
+            },
+            {
+              "type": "className",
+              "value": "android.widget.EditText"
+            },
+            {
+              "type": "xpath",
+              "value": "//android.widget.EditText[@resource-id='com.android.chrome:id/url_bar']"
+            },
+            {
+              "type": "xpath",
+              "value": "//android.widget.EditText[@text='edition.cnn.com']"
+            }
+          ]
+        }
+      ],
+      "actionJson": "{\"command\":\"touchOnElement\",\"x\":\"0.387\",\"y\":\"0.430\"}",
+      "findingElementTimeout": 7129,
+      "isOnKeyboard": false
+    },
+    {
+      "id": "17001",
+      "context": "NATIVE",
+      "actionJson": "{\"command\":\"sendKeys\",\"value\":\"cnn.com\"}",
+      "findingElementTimeout": 4014,
+      "isOnKeyboard": false
+    },
+    {
+      "id": "17002",
+      "context": "NATIVE",
+      "actionJson": "{\"command\":\"press\",\"value\":\"Enter\"}",
```

```suggestion
      "actionJson": "{\"command\":\"press\",\"value\":\"ENTER\"}",
```
Please copy `testSteps` data from java file then run unit test again and check if project can be build